plugins/katello/3.15/installation/index.md

```diff
@@ -29,7 +29,7 @@ Katello does not currently support installation on existing Foreman deployments.
 Katello may be installed onto a baremetal host or on a virtual guest.  The minimum requirements are:
 
 * Two Logical CPUs
-* 8 GB of memory (12 GB highly recommended)
+* 8 GB of memory (12 GB or greater may be required depending on the amount of managed content)
```

Comment:
i think we should likely bump the minimum to 12 and the recommended to 16?

Reply:
Sure, I was considering that too.